lib/api_account.go

```diff
@@ -94,17 +94,23 @@ func GetAccountTransactionsHandler(storage *sebakstorage.LevelDBBackend) http.Ha
 			iterateId := sebakcommon.GetUniqueIDFromUUID()
 			go func() {
 				<-readyChan
+				var iterList []BlockTransaction
 				count := maxNumberOfExistingData
-				iterFunc, closeFunc := GetBlockTransactionsByAccount(storage, address, false)
+				iterFunc, closeFunc := GetBlockTransactionsByAccount(storage, address, true)
 				for {
 					bt, hasNext := iterFunc()
 					count--
 					if !hasNext || count < 0 {
 						break
 					}
-					observer.BlockTransactionObserver.Trigger(fmt.Sprintf("iterate-%s", iterateId), &bt)
+					iterList = append(iterList, bt)
 				}
 				closeFunc()
+
+				for idx := range iterList {
+					bt := iterList[len(iterList)-1-idx]
+					observer.BlockTransactionObserver.Trigger(fmt.Sprintf("iterate-%s", iterateId), &bt)
```

Comment:
Wouldn't it be better to trigger as we iterate, instead of duplicating the list and then iterate on it to trigger ?

Reply:
Iterate is reversed. From the latest to 10 item before. Hence we need to reverse it again.
If there is better idea then let me know